mlflow/tracking/fluent.py

```diff
@@ -182,10 +207,31 @@ def active_run():
     (parameters, metrics, etc.) through the run returned by ``mlflow.active_run``. In order
     to access such attributes, use the :py:class:`mlflow.tracking.MlflowClient` as follows:
 
-    .. code-block:: py
+    .. code-block:: python
+        :caption: Example
+
+        from pprint import pprint
+        import mlflow
 
         client = mlflow.tracking.MlflowClient()
         data = client.get_run(mlflow.active_run().info.run_id).data
+
+        # Extract only user defined tags; skip System tags starting with "mlflow."
+        tags = {k: v for k, v in data.tags.items() if not k.startswith("mlflow.")}
+
+        # Pretty print the dictionary
+        pprint(data.params)
+        pprint(data.metrics)
+        pprint(tags)
+
+    .. code-block:: text
+        :caption: Output
+
+        {'learning_rate': '0.01', 'n_estimators': '10'}
+        {'mse': 2500.0, 'rmse': 50.0}
+        {'engineering': 'ML Platform',
+         'release.candidate': 'RC1',
+         'release.version': '2.2.0'}
```

Comment:
```suggestion
        {'p': '0'}
        {'m': 1.0}
        {'t': '2'}
```

the fixed example code should output this.

Reply:
Will keep it simple. Let's do it!